news/2022-01-16-skinning-contest-tides-of-winter-voting-open.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Skinning Contest: Tides of Winter - Voting Open"
+date: 2022-01-16 12:00:00 +0000
+---
+
+Just as the Winter tides arrive at our doorsteps, we've got the *coolest* skins of all time packed with ice cubes inside a deep frost. Here, come inside and see what's going on!
+
+![](/wiki/shared/news/2022-01-16-skinning-contest-tides-of-winter-voting-open/banner.jpg)
+
+There are a total of **17 freezing entries** delicately crafted by **17 skinners extraordinaire** among yours truly. While they have put forward their best craftsmanship, the task of choosing the coldest falls into your hands! Before casting your votes, be sure to check them all out by yourself in order to appreciate the effort put into them to the fullest.
+
+We have listed all of these majestic icy-cold pieces of artistry inside our resident deep freezer, otherwise known as the one and only [completed skins compendium](https://compendium.skinship.xyz/contests/2). You are highly encouraged to browse around and examine them thoroughly. Though, be sure to wear your gloves, as some of these might be the most frosty skins you'll ever see!
```

Comment:
```suggestion
We have listed all of these majestic icy-cold pieces of artistry inside our resident deep freezer, otherwise known as the one and only [completed skins compendium](https://compendium.skinship.xyz/contests/2). You are highly encouraged to browse around and examine them thoroughly. However, be sure to wear your gloves, as some of these might be the most frosty skins you'll ever see!
```